Automattic/jetpack
# Social Plugin: add Blaze feature

> **Warning**
> Do not review or merge until #31750 is merged.

## Proposed changes:

This is a second take on #28716, now that we have a wp-admin Blaze dashboard since #30103.

This adds 4 pieces of UI to wp-admin for folks using the Social plugin:

1. A "Blaze" link under each published post in Posts > All Posts.
2. A Blaze post-publish panel in the block editor.
3. A Tools > Advertising menu.
4. A new toggle under Jetpack > Social.

**To do**

- [ ] Ensure the module status is kept in sync when used on a site where both Jetpack and Jetpack Social are active.
- [ ] Of note, the toggle is copied from the existing Publicize module toggle. It may be interesting to extract this into a component that can be used by both modules.

### Other information:


## Jetpack product discussion

* 1386-gh-dotcom-forge

## Does this pull request change what data or activity we track or use?

* Yes, it adds 2 new Tracks events on the post editor, when you publish a new post.

## Testing instructions:

Start with a site that runs only the Jetpack Social plugin, that's connected to WordPress.com. Your site must be public.

1. Go to Jetpack > Social and turn the feature on.
2. Click on the button to load the Advertising dashboard.
3. Go to the Posts menu in wp-admin: you should see a "Blaze" link when moving your mouse over published posts.
4. Go to Posts > Add New and write a new post.
6. Upon publishing, you should see a post-publish panel in the post editor sidebar, inviting you to blaze your post.

You'll also want to ensure that the module status is synced with the Jetpack module status.
